client/components/domains/domain-product-price/style.scss

```diff
@@ -18,12 +18,14 @@
 
 	.is-section-signup & {
 		@include breakpoint( '>660px' ) {
+			padding-left: 1em;
 			padding-right: 2em;
 		}
 	}
 
 	.is-section-domains & {
 		@include breakpoint( '>800px' ) {
+			padding-left: 1em;
 			padding-right: 2em;
 		}
```

Comment:
> I haven't looked too closely sorry, but on the face of it, wouldn't this breakpoint be redundant since we're already defining >660px above?

I think it's that these two breakpoints are for when the `domain-product-price` is used in the signup flow, versus being displayed in the domains area within Calypso when you're adding a domain to an existing site (`.is-section-signup` versus `.is-section-domains`). From playing around with it, it feels like the separate breakpoint is to account for the sidebar. I'll leave this as is for now, but happy to make further adjustments in another PR if this looks off!

Reply:
Sorry, I'm blind. Thanks for your patient explanation :) Totally missed the signup vs domains.